css-forms-1/Overview.bs

```diff
@@ -696,6 +696,56 @@ input[type=radio]:checked::checkmark {
     width: 100%;
 }
 ```
+
+## Text fields
+
+```css
+input:not([type]),
+input:is([type=text], [type=search], [type=email], [type=password], [type=tel], [type=url], [type=number]),
+textarea {
+  display: inline-grid;
+  padding-inline: 2px;
+  field-sizing: content;
+}
+
+input:not([type]),
+input:is([type=text], [type=search], [type=email], [type=password], [type=tel], [type=url], [type=number]) {
+  overflow-x: auto;
+  scrollbar-width: none;
+  padding-block: 2px;
+  height: calc(1lh + 2px + 2px);
+  width: calc(20ch + 4px + 2px);
+}
+
+textarea {
+  overflow-y: auto;
+  padding-block: 2px;
+  resize: both;
+  height: calc(2lh + 4px + 2px);
+  width: calc(22ch + 4px + 2px);
```

Comment:
Weird calcs because of `box-sizing: border-box`.

Reply:
note an explicit width/height (even if they match what browsers calculate as their intrinsic size) will break things like auto-stretching within grid layout.

Reply:
Interesting... Not really sure on a solution to that.

I do think field-sizing: content as the default makes sense still but this might make it trickier...

Reply:
Would setting a max-width aswell still break the autostretching? Or does it only break when explicitly set as a width or height?

Reply:
https://github.com/w3c/csswg-drafts/issues/12019#issuecomment-2770409866 per this it seems like calc-size might be the way to go but needs further thinking.